tests/Microsoft.Fx.Portability.MetadataReader.Tests/Tests/BaselineTests.cs

```diff
@@ -64,6 +64,8 @@ private class AssemblyFileClass : IAssemblyFile
             public bool Exists => _file.Exists;
 
             public Stream OpenRead() => _file.OpenRead();
+
+            public bool SkipBinaryIfPackageExists { get { return false; } }
```

Comment:
Please stay consistent with the rest of the file; `SkipBinaryIfPackagesExist => false;`

Reply:
Done